frontend/src/features/leases/list/LeaseFilter/LeaseFilter.tsx

```diff
@@ -7,7 +7,8 @@ import { PropertyFilterOptions } from 'features/properties/filter';
 import { Formik } from 'formik';
 import useLookupCodeHelpers from 'hooks/useLookupCodeHelpers';
 import Multiselect from 'multiselect-react-dropdown';
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useState } from 'react';
```

Comment:
nit: please put all react imports on same line

Reply:
weird, I thought the lint command took care of this.